joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstForStatementsCreator.scala

```diff
@@ -300,10 +300,12 @@ trait AstForStatementsCreator(implicit withSchemaValidation: ValidationMode) { t
       .code(code(usingStmt))
       .lineNumber(line(usingStmt))
       .columnNumber(column(usingStmt))
+    val declAst = Try(createDotNetNodeInfo(usingStmt.json(ParserKeys.Declaration))).toOption match
```

Comment:
do we need to convert `Try()` to Option using `.toOption`?

```
Try(createDotNetNodeInfo(usingStmt.json(ParserKeys.Declaration))) match {
    case Success(declNodevalue) => astForNode(declNodevalue)
    case _                   => Seq.empty[Ast]
}
```

Reply:
Yes we need this, similar type of handling i saw [here](https://github.com/Privado-Inc/joern/blob/f70d6decb8334c2fa08d267dbf4836249f441958/joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstForExpressionsCreator.scala#L538) also. This option is need for case statements. I tried removing it, but build throws error.

Reply:
Yes this also we can do, Actually i saw similar type of handling [here](https://github.com/Privado-Inc/joern/blob/f70d6decb8334c2fa08d267dbf4836249f441958/joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstForExpressionsCreator.scala#L538) where we using `.toOption` instead.

Reply:
done changed.